pkg/network/node/ovscontroller.go

```diff
@@ -203,6 +203,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	otx.AddFlow("table=100, priority=300,udp,udp_dst=%d,actions=drop", vxlanPort)
 	otx.AddFlow("table=100, priority=200,tcp,tcp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
 	otx.AddFlow("table=100, priority=200,udp,udp_dst=53,nw_dst=%s,actions=output:2", oc.localIP)
+	otx.AddFlow("table=100, priority=150,ct_state=+rpl,actions=goto_table:101")
```

Comment:
does this need +trk too?

Reply:
> does this need +trk too?

no, if any other bit is set then `trk` logically has to be set, so `ct_state=+rpl+trk` would be the same as `ct_state=+rpl`